examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,495 @@
+    "trains"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e52dd161-b878-4d6f-b4a0-bda3a6f35967",
+   "metadata": {},
+   "source": [
+    "We can also show this data in a human-readable way."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "071a82d9-9da6-484f-951c-2c0bbc26f0f6",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Northbound 1 to Van Cortlandt Park-242 St, departed origin 11:39:30, Currently STOPPED_AT Christopher St-Sheridan Sq, last update at 12:52:08\n",
+      "Northbound 1 to Van Cortlandt Park-242 St, departed origin 11:45:30, Currently STOPPED_AT Chambers St, last update at 12:51:41\n",
+      "Northbound 1 to Van Cortlandt Park-242 St, departed origin 11:51:30, Currently IN_TRANSIT_TO Rector St, last update at 12:52:23\n",
+      "Northbound 2 to Gun Hill Rd, departed origin 11:11:30, Currently STOPPED_AT 34 St-Penn Station, last update at 12:51:58\n",
+      "Northbound 2 to Wakefield-241 St, departed origin 11:19:30, Currently IN_TRANSIT_TO 14 St, last update at 12:52:26\n",
+      "Northbound 2 to Gun Hill Rd, departed origin 11:27:30, Currently IN_TRANSIT_TO Wall St, last update at 12:52:27\n",
+      "Northbound 2 to Wakefield-241 St, departed origin 11:35:30, Currently STOPPED_AT Atlantic Av-Barclays Ctr, last update at 12:52:05\n",
+      "Northbound 2 to Gun Hill Rd, departed origin 11:43:30, Currently STOPPED_AT President St-Medgar Evers College, last update at 12:51:56\n",
+      "Northbound 2 to Wakefield-241 St, departed origin 11:51:30, Currently IN_TRANSIT_TO Newkirk Av-Little Haiti, last update at 12:52:22\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:05:00, Currently STOPPED_AT 14 St, last update at 12:51:45\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:11:00, Currently STOPPED_AT Chambers St, last update at 12:51:58\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:19:00, Currently INCOMING_AT Wall St, last update at 12:52:26\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:27:00, Currently IN_TRANSIT_TO Hoyt St, last update at 12:52:22\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:35:00, Currently INCOMING_AT Franklin Av-Medgar Evers College, last update at 12:52:25\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:43:00, Currently STOPPED_AT Saratoga Av, last update at 12:51:38\n",
+      "Northbound 3 to Harlem-148 St, departed origin 11:51:30, Currently INCOMING_AT Van Siclen Av, last update at 12:52:06\n"
+     ]
+    }
+   ],
+   "source": [
+    "for train in trains:\n",
+    "    print(train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18cc16fa-686b-4b27-be0d-0015ab28ddcc",
+   "metadata": {},
+   "source": [
+    "We can now check for the times when trains have passed through 34St-Penn Station."
```

Comment:
Minor but this is when trains will pass through the station in the future, not when they have passed through.